src/sentry.js

```diff
@@ -1,11 +1,36 @@
 /* @flow strict-local */
-import { Sentry } from 'react-native-sentry';
-
+import * as Sentry from '@sentry/react-native';
 import config from './config';
 
-if (config.enableSentry) {
-  Sentry.config(config.sentryKey, {
-    deactivateStacktraceMerging: true,
-    ignoreErrors: ['Network request failed'],
-  }).install();
-}
+/** Initialize Sentry into its default configuration. */
+const initSentry = () => {
+  // Under no circumstances should Jest ever execute this function.
+  if (process.env.NODE_ENV === 'test') {
+    throw new Error('initSentry should not be run while testing!');
+  }
+
+  const key = config.sentryKey;
+  if (key != null) {
+    // The DSN is formatted as an `https:` URL. Omit the scheme.
+    const displayKey = `${key.slice(8, 12)}......`;
+    // eslint-disable-next-line no-console
+    console.log(`Sentry key ${displayKey} provided; initializing`);
+
+    Sentry.init({
+      dsn: key,
+      ignoreErrors: ['Network request failed'],
+    });
+  } else {
+    // This is normal behavior when running locally; only builds shipped to the
+    // storefronts will have a Sentry key.
```

Comment:
Well, for once I know exactly how this happened. This actually got fixed for the old PR... but the fix didn't get carried over to the new one, because I didn't pull everything over from the Mac – just a few cherry-picked commits. :man_facepalming: